deploy/crds/noobaa.io_noobaas_crd.yaml

```
@@ -1327,6 +1328,11 @@ spec:
                       type: string
                     type: array
                 type: object
+              manualDefaultBackingStore:
+                description: ManualDefaultBackingStore (optional - default value is
+                  false) if true it will allow the user to delete DefaultBackingStore
```

Maybe change the description to describe the manual treatment of the DefaultBackingStore

---

Sure @liranmauda 
I will change it to :
```suggestion
                description: ManualDefaultBackingStore (optional - default value is
                  false) if true it will allow the user to delete DefaultBackingStore, user needs to delete associated buckets 
                  and update the admin account with new BackingStore in order to delete the DefaultBackingStore
```